ui-components/src/tables/AgTable.tsx

```diff
@@ -207,8 +207,12 @@ const AgTable = ({
 
   return (
     <div className={`ag-theme-alpine ag-theme-bloom ${className || ""}`}>
-      <div className="flex justify-between flex-col md:flex-row">
-        <div className={`flex flex-wrap ${showSearch ? "mb-5" : "hidden"}`}>
+      <div
+        className={`flex justify-between flex-col md:flex-row ${
+          (showSearch || headerContent) && "my-4"
```

Comment:
This is outputting `undefined` when headerContent is undefined and showSearch is false. It's working, but we should make sure nothing is outputted in that case
![image](https://user-images.githubusercontent.com/42942267/198312218-310b8f1b-e5e6-485b-9931-43969f582cb7.png)

Reply:
Fixed 🎾